opsimulator/opsimulator.go

```diff
@@ -118,9 +113,14 @@ func (opSim *OpSimulator) Start(ctx context.Context) error {
 		}
 	}
 
-	opSim.startStartupTasks()
-	if err := opSim.startupTasks.Wait(); err != nil {
-		return fmt.Errorf("failed to start opsimulator: %w", err)
```

Comment:
nit: wdyt about putting the for loop inside a single startupTask? The main point of the startupTasks field was to better separate tasks that are run before the opsim returns ready vs. ones that can run after opsim returns ready.

That way in a separate entrypoint, a caller can just wait on startupTasks to see if opsim is ready

```
opSim.startupTasks.Go(func() error {
    for chain in ...
        addDependencySet
}
```

Reply:
sounds good, I updated the startupTasks to include `configureInteropForChain` and `addDependencies`